README.md

```diff
@@ -38,6 +38,7 @@ _Frameworks for building ASGI web applications._
 - [Quart](https://github.com/pgjones/quart) - A Python ASGI web microframework whose API is a superset of the Flask API. Supports HTTP (incl. SSE and HTTP/2 server push) and WebSockets.
 - [Responder](https://python-responder.org/en/latest/) - A familiar HTTP Service Framework for Python, powered by Starlette. (ASGI 2.0 only, ed.)
 - [Starlette](https://www.starlette.io/) - Starlette is a lightweight ASGI framework/toolkit, which is ideal for building high performance asyncio services. Supports HTTP and WebSockets.
+- [Sanic](https://sanicframework.org/) - Sanic is a Python 3.6+ web server and web framework that's written to go fast. It allows the usage of the async/await syntax added in Python 3.5, which makes your code non-blocking and speedy. Supports HTTP and WebSockets.
```

Comment:
In alphabetical order (see [Contributing](https://github.com/florimondmanca/awesome-asgi/blob/master/CONTRIBUTING.md)), 'Sanic' should go before 'Starlette'. :-)

Reply:
I updated the PR template just now with an extra checkbox to help future contributors catch this at the time of submission.

Reply:
Oops my bad. fixed.